packages/entities/entities-plugins/src/components/free-form/shared/ArrayField.vue

```diff
@@ -47,10 +54,16 @@
         >
           <div class="ff-array-field-item-content">
             <slot
+              v-if="$slots.item"
+              data-autofocus
+              :field-name="String(index)"
```

Comment:
Should we consider receiving a method for constructing the name? For instance:

```ts
const getNameOfField = (idx: number) => {
  return props.generateName(idx) || String(idx)
}
```

Reply:
The field name should be part of the field path, I called it field-name because it conflicts with the slot's name, so it shouldn't be constructed by users.